src/jit/lowerxarch.cpp

```diff
@@ -858,7 +859,7 @@ void Lowering::LowerSIMD(GenTreeSIMD* simdNode)
         }
 
         simdNode->gtFlags |= GTF_SET_FLAGS;
-        simdNode->SetUnusedValue();
+        simdNode->gtType                    = TYP_VOID;
```

Comment:
Do we still need to set `isNoRegCompare` below? This was done to avoid `isLocalDefUse` getting set but since now `IsUnusedValue` is no longer set this appears to be superfluous.